Minio/Exceptions/AccessDeniedException.cs

```diff
@@ -42,9 +41,4 @@ public AccessDeniedException()
     public AccessDeniedException(string message, Exception innerException) : base(message, innerException)
     {
     }
-
-    protected AccessDeniedException(SerializationInfo serializationInfo, StreamingContext streamingContext) : base(
```

Comment:
How is this change (and all other similar changes) became part of this PR?

Reply:
This constructor is obsolete in net8. 
`error SYSLIB0051: 'Exception.Exception(SerializationInfo, StreamingContext)' is obsolete: 'This API supports obsolete formatter-based serialization. It should not be called or extended by application code.' (https://aka.ms/dotnet-warnings/SYSLIB0051)`